router/xgress_edge/fabric.go

```diff
@@ -188,14 +188,6 @@ func (conn *edgeXgressConn) close(notify bool, reason string) {
 
 	log := pfxlog.ContextLogger(conn.Channel.Label()).WithField("connId", conn.Id())
 	log.Debugf("closing message sink, reason: %v", reason)
```

Comment:
should this message travel with this block?

Reply:
no, it's supposed to be at the top. I should fix the text, though

Reply:
fixed